admin/json.go

```diff
@@ -21,6 +21,11 @@ func writeJSONError(w http.ResponseWriter, err *HttpError) {
 	writeJSON(w, response)
 }
 
+func writeJSONOk(w http.ResponseWriter) {
+	response := make(map[string]string)
```

Comment:
I added c816df83a70cbbceeafddc40c645f2c483bc8a61. Is the `== nil` check necessary? It seems like init is only called once, but I just learned about it so I'm not sure. Also should it be a `*[]byte` or is the slice enough because it's pass-by-ref?